grants/accepted_grant_applications.md

```diff
@@ -1,6 +1,8 @@
-# Accepted Grant Applications <!-- omit in toc -->
+# Accepted Grant Applications [WIP!] <!-- omit in toc -->
```

Comment:
Do you want to leave this here on purpose? It's more of a living document now and not a WIP

Reply:
Agree with Alex

Reply:
I thought about phrasing this to make it explicit that this is a living document, but given the document's purpose, it should be obvious to any reader.

Reply:
Those were my thoughts also

Reply:
Sorry, I removed it. The only purpose of it was that teams already reached out to me yesterday and complained about the status of this document.

Reply:
So I quickly added WIP